app/controllers/admin/api/account/proxy_configs_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class Admin::Api::Account::ProxyConfigsController < Admin::Api::BaseController
+  include ApiAuthentication::BySsoToken
+
+  represents :json, entity: ::ProxyConfigRepresenter
+  represents :json, collection: ::ProxyConfigsRepresenter
+
+  clear_respond_to
+  respond_to :json
+
+  rescue_from(ProxyConfig::InvalidEnvironmentError) do
+    render_error 'invalid environment', status: :unprocessable_entity
+  end
+
+  def index
```

Comment:
This endpoint is done for APIcast... should we also document it for APIdocs in order for customers to use it and have support for it ? @3scale/system-ruby

Reply:
https://github.com/3scale/porta/pull/2205